.github/workflows/firebase-hosting-merge-deploy-newm-marketplace.yml

```diff
@@ -0,0 +1,43 @@
+name: Deploy NEWM Marketplace to Firebase Hosting on merge
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - .github/workflows/firebase-hosting-merge-deploy-newm-marketplace.yml
+      - apps/marketplace/**
+      - packages/**
+jobs:
+  build_and_deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: npm install
+      - run: npx nx build marketplace
+        env:
+          NEXT_PUBLIC_APPLE_CLIENT_ID: ${{ secrets.APPLE_CLIENT_ID }}
+          NEXT_PUBLIC_GOOGLE_CLIENT_ID: ${{ secrets.GOOGLE_CLIENT_ID }}
+          NEXT_PUBLIC_GA_STUDIO_ID: ${{ secrets.GA_STUDIO_ID }}
+          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+          NEXT_PUBLIC_ENV: staging
+
+      - uses: FirebaseExtended/action-hosting-deploy@v0
+        id: deploy
+        with:
+          repoToken: "${{ secrets.GITHUB_TOKEN }}"
+          firebaseServiceAccount: "${{ secrets.FIREBASE_SERVICE_ACCOUNT_NEWM_ARTIST_PORTAL }}"
+          channelId: live
+          projectId: "${{ secrets.FIREBASE_PROJECT_ID_NEWM_ARTIST_PORTAL }}"
+          target: marketplace
```

Comment:
Just wanted to confirm, the `NEWM_ARTIST_PORTAL` references are due to the two sites sharing the same Firebase project?